### PR TITLE
Various minor style and grammar improvements to config options descriptions

### PR DESCRIPTION
**Pull request by georgefst**
Sorry, I'm a pedant. There's a lot of room for bikeshedding here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,20 +41,22 @@ We share all bar one of Ormolu's goals:
...
-| `haddock-style`          | `single-line`, `multi-line`, `multi-line-compact` | Whether multiline haddocks should use `-- \|`, `{- \|`, or `{-\|` (single-line haddocks will always use `--` for now) |
-| `let-style`              | `auto`, `inline`, `newline`, `mixed` | How to style `let` blocks, with `auto` using `newline` if the user explicitly puts a newline after `let` and `inline` otherwise, and `mixed` using `inline` if `let` has exactly one binding and `newline` otherwise. |
-| `in-style`               | `left-align`, `right-align` | How to align the `in` keyword with respect to the `let` keyword |
-| `respectful`             | `true`, `false` | Whether to respect user-specified newlines, e.g. in import groupings |
-| `fixities`               | A list of strings | See the "Language extensions, dependencies, and fixities" section below |
+Defaults are in bold.
+
+| Configuration option     | Valid options                                         | Description
+|--------------------------|-------------------------------------------------------|-------------
+| `indentation`            | any non-negative integer (**4**)                      | How many spaces to use as an indent
```
Comment (brandonchinn178) [inline README.md:48]: Can we wrap the numbers in backticks as well?

File: README.md [hunk 2]
```diff
@@ -41,20 +41,22 @@ We share all bar one of Ormolu's goals:
...
+| `record-brace-space`     | `true`, **`false`**                                   | `rec {x = 1}` _vs_ `rec{x = 1}`
+| `newlines-between-decls` | any integer (**1**)                                   | Number of newlines between top-level declarations
+| `haddock-style`          | `single-line`, **`multi-line`**, `multi-line-compact` | Use `-- \|`, `{- \|`, or `{-\|` for multiline haddocks (single-line haddocks always use `--`)
+| `let-style`              | `inline`, `newline`, **`auto`**, `mixed`              | How to style `let` blocks (`auto` uses `newline` if there's a newline in the input and `inline` otherwise, and `mixed` uses `inline` only when the `let` has exactly one binding)
+| `in-style`               | `left-align`, **`right-align`**                       | How to align the `in` keyword with respect to `let`
+| `respectful`             | **`true`**, `false`                                   | Be less aggressive in reformatting input, e.g. keep empty lines in import list
+| `fixities`               | list of strings (**empty**)                           | See the "Language extensions, dependencies, and fixities" section below
```
Comment (brandonchinn178) [inline README.md:59]:
```suggestion
| `fixities`               | list of strings (**`[]`**)                           | See the "Language extensions, dependencies, and fixities" section below
```

**Review by brandonchinn178 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (georgefst) [reply README.md]: Sure, for some reason I was treating it as abstract, thinking we didn't actually this syntax.

Comment (brandonchinn178) [reply README.md]: It's YAML, so json values are valid values

**Pull request merged**